formats/case_study/frontend/examples/archived.json

```diff
@@ -15,9 +16,15 @@
   "details": {
     "change_note": null,
     "tags": {
-      "browse_pages": [],
-      "topics": [],
-      "policies": []
+      "browse_pages": [
```

Comment:
I don't think we should include these formatting changes (and subsequent ones) here as they're not related to the subject of the pull request.